### DISCUSSION TITLE
Github Org-level default preset file naming: default.json or org-inherited-config.json ?

### DISCUSSION THREAD

**Discussion by Djiit** [renovatebot/renovate#30423] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github, Renovate v37.409.1

### Please tell us more about your question or problem

Hello team,

I can't wrap my head around the expected default preset. Documentation mention both default.json and org-inherited-config.json.

I'd like to use Renovate defaults. My org has a renovate-config repo which contains a default preset. Should it be named org-inherited-config.json or default.json ?

I'd be happy to open a PR on the docs when I understand better this, if needed.

Thanks.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Both are valid for different purposes. Are you wanting an implicit shared config (automatically included for every repo regardless of their renovate.json) or an explicit shared config (needs to be extended from renovate.json and will be added to the onboarding config if present)?

And if you don't understand my question above, please just describe in simple terms what you want to achieve with the shared config

**Reply by Djiit** [reply]
Thanks for your quick answer. Both are fine. Let's say implicit. They're going to be able to override it anyway, right ?

**Reply by rarkins** [reply, marked as answer]
Yes but the downside of implicit is that some may not know it's there and be confused why behavior is non-default. The upside of explicit is that it's got maximum clarity to repo users about where the config is coming from.

For implicit you use the inherited config option/name. For explicit the default file name would be default.json

**Reply by Djiit** [reply]
Ok, I'm convinced :).

So preset goes in `org/renovate-config/default.json`. I still have a few questions in this case, just to make sure I unsterdand:
- Self Hosted config `inheritConfig` should be `false` or `true` ? I'd say false.
- New repos will have their config explicitely extending the `org/renovate-config/default.json`, that's it ? e.g. will look like:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["local>org/renovate-config"]
}
```

**Reply by rarkins** [reply]
- false
- yes

**Reply by Djiit** [reply]
THank you !
